tests_e2e/pipeline/templates/execute-tests.yml

```diff
@@ -26,20 +26,23 @@ jobs:
 
       - bash: $(Build.SourcesDirectory)/tests_e2e/pipeline/scripts/execute_tests.sh
         displayName: "Execute tests"
+        continueOnError: true
         env:
           # Add all KeyVault secrets explicitly as they're not added by default to the environment vars
           AZURE_CLIENT_ID: $(AZURE-CLIENT-ID)
           AZURE_CLIENT_SECRET: $(AZURE-CLIENT-SECRET)
           AZURE_TENANT_ID: $(AZURE-TENANT-ID)
           SUBSCRIPTION_ID: $(SUBSCRIPTION-ID)
 
+      - publish: $(Build.ArtifactStagingDirectory)
+        artifact: 'artifacts'
```

Comment:
How should I be accessing the test artifacts/logs?

I thought you were going to the pipeline's published items, but when I go to check the published artifacts on the test run I don't see the same hierarchy you shared on our call:
![image](https://user-images.githubusercontent.com/93676569/213569290-0cf4420d-0071-4b47-98bd-7146dab9a218.png)